db/migrate/20200625180719_create_tests.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTests < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tests do |t|
+      t.string :title
+      t.integer :level, :default => 0, :null => false
```

Comment:
```suggestion
      t.integer :level, default: 0, null: false
```
лучше не использовать "устаревший" синтаксис, там где возможно